webpack.base.config.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
     contentscript: './src/contentscript/index.ts',
     inpage: './src/inpage/index.ts',
     popup: './src/popup/index.tsx',
+    scryptworker: './src/background/workers/scryptworker.js',
```

Comment:
can we access this by defining it via `web_accessible_resources`? from what i gathered thats generally how the extension should access files from the build files.

Reply:
just tried it, doesn't work, I get an uncaught syntax error for the import line. Looks like we need webpack to take care of the importing.

Reply:
Gotcha. Adding it to webpack must allow the node code to be converted.